plugins/jobs/web_client/views/JobStatusSegmentizer.js

```diff
@@ -0,0 +1,37 @@
+import _ from 'underscore';
+import JobStatus from '../JobStatus';
+
+export default class JobStatusSegmentizer {
+    segmentize(jobs) {
```

Comment:
:+1: to moving this.  It isn't a view.  Until we find a use for it elsewhere, it could be moved as just a simple function inside `JobListWidget.js`.